Default to localhost when Host missing (#962)
elastic/apm-agent-ruby: docs/supported-technologies.asciidoc

- Http.rb (v0.6+)
- Faraday (v0.2.1+)

> *Note:* Some libraries like Net/HTTP and Faraday can make requests without specifying a Host.
> When no Host is specified they, and the agent, will fall back to `localhost` as the Host.
> 

Author: ```suggestion
*Note:* These libraries usually assume `localhost` if no `Host` is specified, so the agent does as well. 

```

> 

Reviewer: I think you missed applying the suggested change.
